build.ts

```diff
@@ -11,6 +11,8 @@ import { scss } from './platforms/scss';
 import { json } from './platforms/json';
 import { javascript } from './platforms/javascript';
 import { scssMixin } from './platforms/scss-mixins';
+import { tailwindv4Formatter } from './formats/tailwindv4';
```

Comment:
please rename the tailwindv4 related scripts to `tailwindThemeCss`. Let's avoid hardcoding fw versions into our code